mario/__init__.py

```diff
@@ -50,6 +50,7 @@
 warnings.filterwarnings("ignore", category=RuntimeWarning)
```

Comment:
filtering warnings is not a good practice. 

we need to see where the errors are appearing and solve it from the source. filtering them is not the best solution